### ISSUE TITLE
in iPhone X above models , menu goes to safeArea 

### ISSUE THREAD

**Issue by abbaspour-narjes** [xmartlabs/XLPagerTabStrip#759]
![Screen Shot 2019-10-08 at 11 43 24 AM](https://user-images.githubusercontent.com/31381554/66378688-dea72780-e9c0-11e9-9016-096a0eb530f3.png)

**Comment by mehulvasa0311** [comment]
Same problem. Any solutions?

**Comment by bivant** [comment]
For me it is not clear enough what is wrong on the screenshot. Check the pull requests section, there are some with safe area issues (e.x. https://github.com/xmartlabs/XLPagerTabStrip/pull/756)

**Comment by abbaspour-narjes** [comment]
@bivant your right I should be more specific, here is the  screenshot  of iPhone 6
XLPagerTabStrip should be shown like this 
![photo_2019-11-10_09-26-14](https://user-images.githubusercontent.com/31381554/68539709-e2054880-039c-11ea-8554-f86ca8254631.jpg)

**Comment by bivant** [comment]
Hello @abbaspour-narjes 
Now it more clear. Title "Trailer" is under the notch. Can you attach minimal project where issue is visible? To check the layout/constraints. I assume it depends on layout as not many people encountered it.

**Comment by clementiano9** [comment]
I think this issue still exists and it's affecting ios13 where view controllers are shown modally in a page sheet, the title is not visible even though the top of the page sheet is below the notch, when I change the view controller's presentation to full screen the title appears again. 
The question is, how do we support the new ios page sheet style? @bivant

**Comment by FaizalMalik** [comment]
Yes , I am also facing the same issue..! In Demo the view controller is presented so there is no issues.
<img width="445" alt="Screen Shot 1441-04-26 at 5 29 32 PM" src="https://user-images.githubusercontent.com/32692684/71363329-d2873b00-25a9-11ea-94ce-8d9992844a9f.png">

**Comment by mohamedghonemi** [comment]
One workaround I found is to use separate view for the buttonBarView (Empty ButtonBarView) and attach it to the outlet in the controller. then you can place it right in the view .
